bin/gwdetchar-lasso-correlation

```diff
@@ -808,15 +844,35 @@ def generate_cluster(input_,):
                             .replace('_', '-', 1),
                     gpsstub)
                 clustertab.write(plot7_list, format='ascii', overwrite=True)
+                colormaps = [cmap(i) for i in numpy.linspace(0, 1, 21)]
             else:
                 cluster_range = len(current_cluster)
-
+                clustertab = (
+                    Table(data=(zip(*current_cluster)[3],
+                                zip(*current_cluster)[2]),
+                          names=('Channel', 'Pearson Coefficient')))
+                plot7_list = '%s_CLUSTER_LIST-%s.txt' % (
+                    re_delim.sub('_', str(currentchan))
+                            .replace('_', '-', 1),
+                    gpsstub)
+                clustertab.write(plot7_list, format='ascii', overwrite=True)
+                colormaps = [cmap(i) for i in
+                             numpy.linspace(0, 1, cluster_range+1)]
```

Comment:
@uberpye Just for conciseness, I think we could bring lines 838-846 / 850-858 outside the `if`/`else` so we only have that block once, but leave the `cluster_range` and `colormaps` initializations inside

Reply:
sounds good, now done. as it is a small change a squashed it in to the earlier commit. now to rebase against upstream/master